app/models/schema/plum/hyrax.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+##
+# Defining the attributes for all Resources within Plum
+module Schema
+  module Plum
+    ##
+    # Defines the attributes migrated from Hyrax core
+    module Hyrax
```

Comment:
I am wary of naming any of our modules or classes "Hyrax" — mostly for symbolic reasons and to avoid confusion if/when Hyrax support for Valkyrie is undertaken.

I also think it would be nice to split up the terms by source vocabulary: many of these are from DC/DCTerms or MARC Relators.